src/IdBrokerClient.php

```diff
@@ -3,13 +3,29 @@
 
 use Exception;
 use GuzzleHttp\Command\Result;
+use IPBlock;
 
 /**
  * IdP ID Broker API client implemented with Guzzle.
  */
 class IdBrokerClient extends BaseClient
 {
     /**
+     * The key for the constructor's config parameter that refers
+     * to the trusted IP ranges.
+     */
+    const TRUSTED_IPS_CONFIG = 'trusted_ip_ranges';
+
+    /**
+     * The list of trusted IP address ranges (aka. blocks).
+     *
+     * @var IPBlock[]
+     */
+    private $trustedIpRanges = [];
+
```

Comment:
Do you want this new config to default to True.  If not, you still have the issue of a typo disabling this feature, right?